coderd/database/migrations/000174_add_user_theme_preference.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users ADD COLUMN "theme_preference" text;
```

Comment:
We tend to try and avoid `null` columns. Can the empty string be sufficient here instead of the null value?

```suggestion
ALTER TABLE users ADD COLUMN "theme_preference" text NOT NULL DEFAULT '';
```